docs/modules.rst

```diff
@@ -28,7 +29,53 @@ The best practice is to do the following:
 * Create a directory for your module inside one of the repositories.
 * Make sure your directory is a valid Python package (do not use spaces, make sure every directory in the path as a ``__init__.py`` file).
 * Create a python file inside your directory for your module.
-* Inside your python file, create a class that inherits from :class:`fame.core.module.ProcessingModule`, :class:`fame.core.module.ReportingModule`, :class:`fame.core.module.ThreatIntelligenceModule` or :class:`fame.core.module.AntivirusModule`.
+* Inside your python file, create a class that inherits from :class:`fame.core.module.PreloadingModule`, :class:`fame.core.module.ProcessingModule`, :class:`fame.core.module.ReportingModule`, :class:`fame.core.module.ThreatIntelligenceModule` or :class:`fame.core.module.AntivirusModule`.
+
+Writing a Preloading module
```

Comment:
Could you explain that a little bit? In my opinion (and compared to in what order the modules will be executed) it makes more sense to put Preloading before Processing simply because the former is executed before the latter.

Reply:
Based on execution order, you are correct, but I think that most people looking to develop modules are looking for Processing Modules. 

In my opinion, it is the main kind of modules so it should always come first.

Reply:
Agreed